pkg/services/featuremgmt/registry.go

```diff
@@ -168,5 +168,10 @@ var (
 			State:           FeatureStateAlpha,
 			RequiresDevMode: true,
 		},
+		{
+			Name:        "resourcePickerForMetrics",
+			Description: "New UI for Azure Monitor Metrics Query",
+			State:       FeatureStateAlpha,
```

Comment:
what about setting `RequiresDevMode: true` so we make sure nobody uses it before it's ready? Also we can use `FrontendOnly: true` but I am not really sure what that does.